### ISSUE TITLE
Example in Java?

### ISSUE THREAD

**Issue by raffian** [developerforce/SalesforceDurablePushTopicDemo#3]
Any plans on providing an example in Java? We're exploring durable subscriptions using Apache Camel and want to customize the connector to support durable streaming,

Best,
Raffi

**Comment by john-brock** [comment]
Yes - please check out https://github.com/developerforce/StreamingReplayClientExtensions/blob/master/java/CometDReplayExtension.java for a Java client example. Hope this helps - let us know.


On Feb 23, 2016, at 6:42 PM, Raffi Basmajian <notifications@github.com<mailto:notifications@github.com>> wrote:


Any plans on providing an example in Java? We're exploring durable subscriptions using Apache Camel and want to customize the connector to support durable streaming,

Best,
Raffi

-

**Comment by raffian** [comment]
Nice, thank you.

What is <V>?....
